script.module.resolveurl/lib/resolveurl/plugins/upbaam.py

```diff
@@ -24,8 +24,8 @@
 
 class UpBaamResolver(ResolveUrl):
     name = 'UpBaam'
-    domains = ['upbaam.com', 'cdnupbom.com']
-    pattern = r'(?://|\.)((?:upbaam|cdnupbom)\.com)/([0-9a-zA-Z]+)'
+    domains = ['upbaam.com', 'cdnupbom.com', 'uupbom.com']
+    pattern = r'(?://|\.)((?:upbaam|cdnupbom|uupbom)\.com)/([0-9a-zA-Z]+)'
```

Comment:
Can be done better
`    pattern = r'(?://|\.)((?:cdn)?u*pb[ao]*m)\.com)/([0-9a-zA-Z]+)'`

Reply:
sorry but i just tested this pattern its not working with `uupbom.com`
can you check it again `https://cima120.uupbom.com/07uc2af5jtof`
the pattern i have pushed is working fine with it